command.php

```diff
@@ -178,7 +188,7 @@ private function choices_theme() {
 				$themes[ $id ] = $theme->get( 'Name' );
 			}
 
-			return cli\menu( $themes, wp_get_theme()->template, 'Choose a theme' );
+			return cli\menu( $themes, get_stylesheet(), 'Choose a theme' );
```

Comment:
@anhskohbo Not sure if you knew this, but the default name was set to `template` which meant if the current theme was a child theme its parent theme would be selected. This did not seem right to me so I changed it to be the stylesheet.